compiler/resolution/functionResolution.cpp

```diff
@@ -9542,9 +9569,8 @@ static void resolveAutoCopyEtc(AggregateType* at) {
   // resolve autoDestroy
   if (autoDestroyMap.get(at) == NULL) {
     FnSymbol* fn = autoMemoryFunction(at, "chpl__autoDestroy");
-
-    fn->addFlag(FLAG_AUTO_DESTROY_FN);
-
+    if (fn)
+      fn->addFlag(FLAG_AUTO_DESTROY_FN);
```

Comment:
When `--minimal-modules` is used, `chpl__autoDestroy` is not defined, and so `fn` is NULL. We need to guard against dereferencing a `NULL` `fn` to prevent a segFault.